### PR TITLE
Us25 physics performance optimization (sprint demo fixes)

### PR DESCRIPTION
**Pull request by Wessebanan**
Some stuff. :)

### CODE REVIEW

**Comment by Wessebanan** [comment]
poyo

**Comment by Wessebanan** [comment]
kridley

File: Game/GameMain/main.cpp [hunk 1]
```diff
@@ -165,7 +165,7 @@ int main()
 
 			if (GetAsyncKeyState(VK_SPACE) && start_once)
 			{
-				{
+			/*	{
 					ecs::events::PlayMusic m_event;
 					m_event.audioName = AudioName::CC_TEST_SONG;
 					ecs.createEvent(m_event);
```
Comment (NistBox) [inline Game/GameMain/main.cpp]: If these are not used anymore, remove it.

**Comment by Wessebanan** [comment]
pixel 4096
width 90
height 60

**Pull request merged**